homeassistant/components/velbus/quality_scale.yaml

```diff
@@ -0,0 +1,70 @@
+rules:
+  # Bronze
+  action-setup: todo
+  appropriate-polling:
+    status: exempt
+    comment: |
+      This integration does not do polling.
+  brands: done
+  common-modules: done
+  config-flow-test-coverage: done
+  config-flow: todo
```

Comment:
We should not have a user configurable name in the config fow. Not a blocker, just wanted to mention it

Reply:
what name should be selected?
some people will need more then 1 of these, so they will need to know what entry is what connection